### ISSUE TITLE
Windows 10 platform has 4 FPS 

### ISSUE THREAD

**Issue by Fatihcill** [RobLoach/raylib-tileson#3]
i tried your code in 2 platforms. it correctly runs in linux with 60 fps. but when I tried your code in windows the fps downs about 4. 
btw i also changed this lines because i had got an error in Windows 10 (as I said your code runs correctly in linux).
  raylib-tileson.cpp line 84;
  `LoadTiledImage(data, baseDir, tileset.getImage(), tileset.getTransparentColor());` to 
  `LoadTiledImage(data, baseDir, tileset.getImage().string(), tileset.getTransparentColor());`

  raylib-tileson.cpp line 84;
  `LoadTiledImage(data, baseDir, tileset.getImage(), tileset.getTransparentColor());` to 
  `LoadTiledImage(data, baseDir, tileset.getImage().string(), tileset.getTransparentColor());`
  
  raylib-tileson.cpp line 120;
 `std::string image = tileset->getImage();` to
 `std::string image = tileset->getImage().string();`

  the error was;
  `note: in passing argument 3 of 'void LoadTiledImage(RaylibTilesonData*, const char*, const string&, tson::Colori)'
  [build]    33 | void LoadTiledImage(RaylibTilesonData* data, const char* baseDir, const std::string& imagepath, tson::Colori transparentColor) {`

**Comment by RobLoach** [comment]
Thanks a lot, Fatihcil16! Would you mind submitting a Pull Request for the change? If not, there's no problem on my making those changes. Thanks again.

**Comment by Fatihcill** [comment]
I sent the pull request to fix the problem on Windows; however, I have not been able to fix the FPS problem. I launched the example in debug and release mode and I realized it didn't have any problems when I launched it in release mode or even when I launched it normally from explorer. The FPS goes goes down only when I launch it in debug mode. Is this natural or do you think there's something wrong going on?

**Comment by RobLoach** [comment]
In Debug mode, this are a lot more cycles going into checking for memory leaks, so I'm not too surprised. Is DEBUG mode the default on Windows?

**Comment by Fatihcill** [comment]
Yes, it's. That's why I'm so confused xdd. I am developing my app in release mode now.
